_includes/header.html

```diff
@@ -16,24 +16,24 @@
                     <li>
                         <a href="/" title="There's no place like Home...">Home</a>
                     </li>
+                    <li>
+                        <a href="/docs/getting-started" title="Our Getting Started guide">Get Started</a>
```

Comment:
> I'm guessing that you changed this to "Get Started" to make it shorter. 

Yeah, that and more declarative as opposed to passive.

> That's fine, but it just strikes me as unusual or awkward for some reason. Not sure how to explain it exactly - it's probably just me. Saying "Start Here" seems a bit more natural to me though.

I get what you're trying to say, and I feel the same. I like the "start here" idea.

> Another thing to consider is that most sites that offer software have a "Download" or "Install" link. I know when I am visiting a new software site, this is what I am looking for.

I've been contemplating the same problem. I expect the lack of a "download" button is disconcerting to most users.

> What do you think about creating a new page just for downloads? ...

I like the idea as you describe it. That would also minimize people downloading the wrong image (which, despite my best efforts with button size and placement, still happens). The full form of that page does depend on a finalized whole-page platform selection mechanism, as you mentioned, so we might need to start this without the nice links and then build on it later.

Do you think I should add this now for the current PR in a partial form, or wait until after I get the rest of it restructured and I can come back to platform selection?